### ISSUE TITLE
CAMM ER

### ISSUE THREAD

**Issue by Venom63it** [PygmalionOfCyprus/cmo-db-requests#1096]
### DB Selector

DB3K

### Baseline

Sea Ceptor [CAMM(M)}

### Hypothetical

No

### Name

CAMM ER

### Length (m)

4.2

### Width/Span (m)

.45

### Diameter (m)

.19

### Weight (kg)

160

### Range (nm)

0.2-32

### Target Altitude / Depth (ft)

30ft AGL- 30000 ft AGL

### Warhead

10Kg frag

### Targets

Aircraft, Helicopter, Missile, Surface Vessel

### Sensors

1x Active radar Seeker 15 nm

### Weapon Codes

Anti-Air Dogfight (High Off-Boresight), Capable vs Seaskimmer

### Comments

ER version of CAMM developped by MBDA Italy for Italian Navy, Army and Airforce.
Basically is the CAMM with a booster produce by Avio
Aeronautica Militare Italiana will use on AA battery  MAADS with KRONOS 3D LAND, 
Esercito Italiano) will use on AA battery GRIFO with Rhainmetall radar  X-TAR 3D 

### Sources

https://www.mbda-systems.com/product/camm-er/
https://www.avio.com/it/propulsione#sect-238
https://ukdefencejournal.org.uk/mbdas-camm-er-missile-completes-major-milestone/

**Comment by PygmalionOfCyprus** [comment]
Added w/ #1098
